locales/en.json

```diff
@@ -98,6 +98,12 @@
     "yesConfirm": "Yes, I Confirm",
     "no": "No"
   },
+  "RemoveVcWarningOverlay": {
```

Comment:
have we added these  literals in all other locales ?

Reply:
Adding in the other locales